src/PIL/ImageFilter.py

```diff
@@ -442,6 +442,15 @@ def transform(self, callback, with_normals=False, channels=None,
                           target_mode=target_mode or self.mode,
                           _copy_table=False)
 
+    def __repr__(self):
+        r = ["{0} from {1}".format(self.__class__.__name__,
```

Comment:
In all these, we can have simple formatters without numbering: https://pyformat.info/#simple

Reply:
Done